test/e2e/components/vl-proza-message.js

```diff
@@ -1,7 +1,91 @@
 const { VlElement } = require('vl-ui-core');
+const { By, Key } = require('selenium-webdriver');
 
-class VlProzaMessage extends VlElement {  
+class VlProzaMessage extends VlElement {
 
+    async _getWysiwyg() {
+        return this.findElement(By.css('#wysiwyg'));
+    }
+
+    async _waitUntilEditable() {
+        return this.driver.wait(async () => {
+            return this.isEditable();
+        }, 5000);
+    }
+
+    async getText() {
+        const wysiwyg = await this._getWysiwyg();
+        return this.driver.executeScript('return arguments[0].innerText', wysiwyg);
```

Comment:
Lukt wysiwyg.getAttribute('innerText') of iets dergelijks niet?

Reply:
Aangepast.